#! /bin/sh /usr/share/dpatch/dpatch-run
## 30-disable-start-by-default.dpatch by Mario Izquierdo (mariodebian) <mariodebian@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: No start by default

@DPATCH@
diff -urNad p910nd-0.92~/p910nd.conf p910nd-0.92/p910nd.conf
--- p910nd-0.92~/p910nd.conf	2008-01-02 12:47:01.000000000 +0100
+++ p910nd-0.92/p910nd.conf	2009-02-08 18:21:16.000000000 +0100
@@ -2,3 +2,7 @@
 P910ND_NUM=""
 # Additional daemon arguments, see man 8 p910nd
 P910ND_OPTS=""
+
+
+# Debian specific (set to 1 to enable start by default)
+P910ND_START=0
